crypto3/libs/algebra/include/nil/crypto3/algebra/algebra.pch

```diff
@@ -0,0 +1,13 @@
+#include <nil/crypto3/algebra/curves/pallas.hpp>
```

Comment:
auto generated `*.pch `should be ignored, right?

Reply:
This whole PR is yet highly experimental